wikidict/render.py

```diff
@@ -268,7 +274,7 @@ def find_all_sections(code: str, locale: str) -> List[Tuple[str, wtp.Section]]:
                 )
             )
 
-    def section_title(title: str) -> str:
+    def section_title(title: str, locale='de') -> str:  
```

Comment:
I believe it would be better to add "ru" to the list instead ?

Reply:
I can't remember what I was doing here, I think it is just an error on my part. I removed it.